app/src/main/java/com/example/memeitupapp/ui/gridmemes/view/GridMemesFragment.kt

```diff
@@ -34,7 +34,7 @@ class GridMemesFragment : Fragment() {
         super.onViewCreated(view, savedInstanceState)
 
         gridMemesViewModel = ViewModelProvider(this, MemeViewModelFactory.viewModelFactory {
-            GridMemesViewModel(GridMemesModel(MemeService()))
+            GridMemesViewModel(GridMemesModel(com.globant.data.service.MemeService()))
```

Comment:
same,  this is not necessary **com.globant.data.service.**

Reply:
Refactored!